lib/etl/mapper.ts

```diff
@@ -26,47 +26,71 @@ export default class WarzoneMapper{
         return dataObject;
     }
     static mapGamerMatch(match, queryGamer){
+        let {playerStats, player} = match;
+        let missionsByType = player.brMissionStats?.missionStatsByType;
```

Comment:
@fjaxyu  the nullsafe . access operator is pretty amazing

Reply:
So wild, I honestly wish this was the default behavior of JS.